docs/tutorials/TUTORIAL_02_CONFIGURING_SETTINGS_JSON.md

```
@@ -0,0 +1,67 @@
+# <img src="../assets/logo.png"> Tutorial 2 – Configuring settings.json
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)